jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -380,6 +379,40 @@ with what features/services are supported.
             <reference id="serverless.application.deploy"/>
             <separator/>
         </group>
+
+        <group id="aws.toolkit.s3viewer.toolbar.pt1">
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.DownloadObjectAction"/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.UploadObjectAction"/>
+            <separator/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.ViewObjectVersionAction"/>
+            <separator/>
+        </group>
+
+        <group id="aws.toolkit.s3viewer.toolbar.pt2">
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.NewFolderAction"/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.RenameObjectAction" use-shortcut-of="RenameElement"/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.DeleteObjectAction" use-shortcut-of="$Delete"/>
+            <separator/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.RefreshTreeAction"/>
+        </group>
+
+        <group id="aws.toolkit.s3viewer.toolbar">
+            <reference ref="aws.toolkit.s3viewer.toolbar.pt1"/>
```

Comment:
something more descriptive that `pt1`/`pt2`?

Reply:
I couldnt think of a name -.-

Reply:
`aws.toolkit.s3viewer.object.transfer`
`aws.toolkit.s3viewer.object.management`

but then viewing objects isn't really related to transfer...

Reply:
Broke it up into more groups, what you think?